java/core/src/main/scala/io/pdal/PointView.scala

```diff
@@ -0,0 +1,77 @@
+package io.pdal
+
+import java.nio.ByteBuffer
+
+class PointView extends Native {
+  def getPackedPointWithMetadata(idx: Long, metadata: String, schema: String): PackedPoints =
+    getPackedPoint(idx).copy(metadata = metadata, schema = schema)
+  def getPackedPoint(idx: Long): PackedPoints = getPackedPoint(idx, layout.dimTypes())
```

Comment:
`PackedPoints` is a generic name for `Array[Byte]` and all metadata around it, to perform operations on point(s) in JVM memory, so it can be a point with a certain dims number.

The main difference between `PointView` and `PackedPoints` is that `PointView` works as a "proxy class" and works using C++ memory and functions.